src/openforms/appointments/utils.py

```diff
@@ -309,3 +309,10 @@ def get_confirmation_mail_suffix(submission: Submission) -> str:
         return ""
 
     return _("(updated)")
+
+
+def get_appointment(submission: Submission) -> Appointment | None:
```

Comment:
I don't like that this `utils` module is now a mess of old, deprecated stuff and new things that are okay to re-use, but I suppose it's the best we can do for now.